scripts/tasks/babel.ts

```diff
@@ -0,0 +1,50 @@
+import { transformAsync } from '@babel/core';
+import * as glob from 'glob';
+import fs from 'fs';
+import { logger } from 'just-task';
+import path from 'path';
+
+const EOL_REGEX = new RegExp(/\r?\n/g, 'g');
```

Comment:
nit, I think you can just do this now:
```suggestion
const EOL_REGEX = /\r?\n/g;
```